### PR TITLE
public: use prod-playbooks tag for redirect

### PR DESCRIPTION
**Pull request by joshmoore**
Also include example inline

See: https://github.com/openmicroscopy/ome-documentation/pull/1835

### CODE REVIEW

File: omero/sysadmins/public.rst [hunk 1]
```diff
@@ -267,4 +267,10 @@ link to the correct group and Project in OMERO, just as the
 DOI above does. Redirects need to be set in the
 `NGINX <http://nginx.org/>`_ component of the OMERO.web installation
 dedicated to publication workflows. You can find our configuration for this
-example `here on GitHub <https://github.com/openmicroscopy/prod-playbooks/blob/2c73f0482bb87dc67e8d011d087ae591ad0bd836/nightshade-webclients.yml#L172>`_.
+example `here on GitHub <https://github.com/openmicroscopy/prod-playbooks/blob/2018-01/nightshade-webclients.yml#L181-L183>`_:
```
Comment (hflynn) [inline omero/sysadmins/public.rst]: https://ci.openmicroscopy.org/view/Docs/job/OMERO-DEV-merge-docs/994/console is complaining about this:

`docs/src/omero/sysadmins/public.rst.rst:266:broken link: https://github.com/openmicroscopy/prod-playbooks/blob/2018-01/nightshade-webclients.yml#L181-L183 (Anchor 'L181-L183' not found)`

Comment (joshmoore) [reply omero/sysadmins/public.rst]: Link checker bug.... I'll roll back to a single line. 😦  cc: @kennethgillen

**Comment by hflynn** [comment]
Merge build looks good this time thanks

File: omero/sysadmins/public.rst (hunk 1 above)
Comment (kennethgillen) [reply omero/sysadmins/public.rst]: 🤕

**Pull request merged**